docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1896,17 +1919,48 @@ async function getStats() {
   if (meeting) {
     console.log('Stats for all video panes:');
     // extract the stats for each video pane
-    Object.values(allVideoPanes).forEach((paneGroup) => {
+    for (const [groupId, paneGroup] of Object.entries(allVideoPanes)) {
       Object.values(paneGroup).forEach(async (videoPane) => {
         const stats = await getStatsForVideoPane(meeting, videoPane);
 
-        console.log(`stats for video pane "${videoPane.remoteMedia.id}": `, stats);
+        console.log(`stats for ${groupId} video pane "${videoPane.remoteMedia.id}": `, stats);
```

Comment:
this is not strictly part of this feature, but just a small improvement that I found useful when debugging things

Reply:
I am not sure how the stats looks like in the console, Are we showing the stats on the video pane ?

Reply:
no this is just for the console, they look fine in the console - I find it useful for debugging. Maybe in the future we can show something on the panes, but there is a lot of stats so it's hard to fit them all.